<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1212</width>
    <height>792</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Triangular</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>1151</width>
      <height>621</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>110</width>
      <height>110</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>22</width>
      <height>22</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <property name="iconSize">
     <size>
      <width>48</width>
      <height>48</height>
     </size>
    </property>
    <property name="elideMode">
     <enum>Qt::ElideNone</enum>
    </property>
    <property name="documentMode">
     <bool>false</bool>
    </property>
    <property name="tabsClosable">
     <bool>false</bool>
    </property>
    <property name="movable">
     <bool>false</bool>
    </property>
    <property name="tabBarAutoHide">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="icon">
      <iconset>
       <normaloff>../../../sundry/icons/iconsvg.png</normaloff>../../../sundry/icons/iconsvg.png</iconset>
     </attribute>
     <attribute name="title">
      <string>网络训练</string>
     </attribute>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>340</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>训练网络</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../../sundry/icons/button.png</normaloff>
        <normalon>../../../sundry/icons/button1.png</normalon>
        <selectedoff>../../../sundry/icons/button1.png</selectedoff>../../../sundry/icons/button.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>731</width>
        <height>331</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../../sundry/pic/app1.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="icon">
      <iconset>
       <normaloff>../../../sundry/icons/icon_video.png</normaloff>../../../sundry/icons/icon_video.png</iconset>
     </attribute>
     <attribute name="title">
      <string>姿态预估</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>510</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>训练网络</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../../sundry/icons/button.png</normaloff>
        <normalon>../../../sundry/icons/button1.png</normalon>../../../sundry/icons/button.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>581</width>
        <height>411</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../../sundry/pic/app2.png</pixmap>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>510</y>
        <width>211</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>结果</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>10</y>
        <width>541</width>
        <height>421</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../../sundry/pic/pic.jpg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>811</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>新宋体</family>
      <pointsize>28</pointsize>
      <underline>false</underline>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>基于人体骨骼姿态的姿势识别</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="margin">
     <number>4</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1212</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
